### DISCUSSION TITLE
Differences in the statamic/configs between a fresh install and a subsequent `php please site:clear`.

### DISCUSSION THREAD

**Discussion by helloDanuk** [statamic/cms#9345] [Q&A] [answered]
Hi!

I noticed that there are differences in a few `/config/statamic`-files between a fresh installation via the command `statamic new loremipsum` and a subsequent `php please site:clear`.

`php please site:clear` removes some configs and adds new ones. Shouldn't the configs on a fresh page be more or less identical? Whether I delete the page content with site:clear or not? I list all the differences here from the config/statamic folder.

Specially in `forms.php`, `stache.php` and `users.php` are some tricky ones for me. As I come from a design background, I cannot understand the differences in the three files with my technical level. Should I go with the ones from a fresh install or with the ones after a site:clear command?

Thank you for a bit of clarity on this.

Here are all changes coming form a fresh install via CLI (v4.44) and after a site:clear command:

**/statamic/antlers.php**
<img width="775" alt="antlers" src="https://github.com/statamic/cms/assets/8265902/6990f223-f345-4909-8f6f-8e46b95b3798">

**assets.php**
<img width="750" alt="assets" src="https://github.com/statamic/cms/assets/8265902/0dd6d0e5-3cc6-48ca-aa6f-f58df7115759">

**cp.php**
<img width="750" alt="cp" src="https://github.com/statamic/cms/assets/8265902/2dbb235b-2272-4b55-9a37-794775ec9566">

**forms.php**
<img width="750" alt="forms" src="https://github.com/statamic/cms/assets/8265902/923693f6-2b68-4374-bf98-276a3f60e1b4">

**git.php**
This is a obvious one, assets container gets removed.
<img width="750" alt="git" src="https://github.com/statamic/cms/assets/8265902/82a2f585-ec67-4491-9f8c-5d65be7b01cd">

**markdown.php**
Doesn't exist in `statamic/statamic`

**routes.php**
<img width="751" alt="routes" src="https://github.com/statamic/cms/assets/8265902/28bc0171-218a-4ea2-a739-2dcc65faea20">

**stache.php**
<img width="750" alt="stache" src="https://github.com/statamic/cms/assets/8265902/8ecc7b52-b8d5-4b5a-b626-ae8f1fe6246f">

**system.php**
<img width="750" alt="system" src="https://github.com/statamic/cms/assets/8265902/a5770c10-7452-43c4-b14d-2a017298fa32">

**users.php**
<img width="750" alt="users" src="https://github.com/statamic/cms/assets/8265902/cecfe8ca-37f5-40b3-881d-07720643aee5">

**Comment by jasonvarga** [comment, marked as answer]
> Shouldn't the configs on a fresh page be more or less identical? 

Yes. It looks like we have a few files that need to be updated within the `site:clear` command.

> Should I go with the ones from a fresh install or with the ones after a site:clear command?

Go with what you see in a fresh site. (The statamic/statamic repo)

https://github.com/statamic/statamic/tree/4.x/config/statamic

**Reply by helloDanuk** [reply]
Thanks!
